bedrock/firefox/templates/firefox/mobile-en.html

```diff
@@ -0,0 +1,218 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% from "macros.html" import google_play_button, send_to_device with context %}
+{% from "macros-protocol.html" import feature_card, hero, call_out_compact with context %}
+
+{% add_lang_files "firefox/sendto" %}
+
+{% extends "firefox/base/base-protocol.html" %}
+
+{% block page_css %}
+  {{ css_bundle('firefox-mobile-protocol') }}
+{% endblock %}
+
+{% block page_title %}{{ _('Mobile browsers for iOS and Android') }} | {{ _('Firefox') }}{% endblock %}
+{% block page_desc %}{{ _('Download Firefox and Firefox Focus for mobile and tablet.') }}{% endblock %}
```

Comment:
Firefox Focus is no longer offered on this page, so should it be here in the description?